doc/README.md

```diff
@@ -28,6 +28,8 @@
 
 [Contributing to status-go](status-go-changes.md)
 
+[Malli schemas (recorded demo)](https://drive.google.com/file/d/1orqa8Ky2mVqGJCLksgRF7F5hqjurMxt0/view)
```

Comment:
This link is private and probably of no use to folks that do not have a status email address.
Should we consider uploading this presentation to Status Youtube and then link it here ?
Also a separate link to PPTs will also be nice

Reply:
I've asked in Logos Discord

Reply:
We can post on youtube, I'm waiting to hear from @ilmotta on his thoughts 😅

Reply:
@siddarthkay @vkjr it's fine by me. I think the presentation doesn't contain anything egregiously wrong and if it helps anyone why not share, right? I'll add the slides to the same Status Demo directory in G. Drive. Thanks guys :)

Reply:
Thanks @ilmotta : This will be uploaded Monday next week.

Reply:
here @vkjr -> https://www.youtube.com/watch?v=SlRio70aYVI